Tests/FaunaDBTests/ClientTests.swift

```diff
@@ -1,5 +1,6 @@
 import XCTest
-import FaunaDB
+
+@testable import FaunaDB
```

Comment:
It's best to keep this suite with customer face API only.
We should be able to simulate echo with public api only. Even if the assert is not explicit.